.github/workflows/new_prefix_pr.yml

```diff
@@ -14,21 +14,47 @@ jobs:
         python-version: [ "3.12" ]
     steps:
       - uses: actions/checkout@v2
+      - name: Delay to ensure labels are attached
+        run: sleep 10
+      - name: Check Issue Labels
+        id: check_labels
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const issue = context.payload.issue;
+            const hasRequiredLabels = issue.labels.some(label => label.name === "New") && issue.labels.some(label => label.name === "Prefix");
+            core.setOutput("hasRequiredLabels", hasRequiredLabels ? 'true' : 'false');
+      - name: End Workflow if Labels are Missing
+        if: steps.check_labels.outputs.hasRequiredLabels == 'false'
+        run: |
+          echo "Issue does not have 'New' and 'Prefix' labels. Ending workflow."
+          exit 0
```

Comment:
If we exit the workflow here, is it still necessary to check below for 
```
if: steps.check_labels.outputs.hasRequiredLabels == 'true'
```
or can those be omitted?

Reply:
Omitting the conditionals does not actually halt the workflow from running for non-prefix issues if that is the goal.

Omitting conditionals:
![image](https://github.com/user-attachments/assets/fe9934a7-cfdb-4a84-a4f0-1a3c3c4c8cb5)

Leaving conditionals in:
![image](https://github.com/user-attachments/assets/08f2fa4d-0918-49ac-b7ed-ef2751907023)